README_ANDROID.md

```diff
@@ -4,6 +4,25 @@
 
 ### Android SDK/NDK
 
+When building the engine, the build system tries to find the sdk in this order:
+
+* `$DYNAMO_HOME/ext/SDKs/android-sdk` -  This is mainly for the Defold team and their CI
+* `$ANDROID_HOME` - Set this path to point to your android studio sdk installation (e.g. `ANDROID_HOME=~/Library/Android/sdk` is the default installation on macOS)
```

Comment:
Currently only tested on macOS. SHould work the same on Linux I think.
On Windows, we should double check that Android Studio creates an ANDROID_HOME environment variable.